### ISSUE TITLE
Syncronization with facebook/gmail or something

### ISSUE THREAD

**Issue by huleboer** [devconfcz/devconf-cz-android#2]
Some login feature with sync of which presentations I'm going to.. Then I can use either the web interface or my android pad to set up where I'm going and use my phone while at devconf

**Comment by jridky** [comment]
yes, i was thought about it yesterday, but right now I will not probably implement it for this year (depends on time I will have for this), so it will be part of app for next year.
